body.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Should this file be here at the project root directory?

Reply:
AHHAHA omg this should definitely not be here. Thanks for noticing